packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view-matrix.stories.ts

```diff
@@ -0,0 +1,64 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
+import type { Meta, StoryFn } from '@storybook/html';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../../../../utilities/tests/matrix';
+import { createMatrixThemeStory } from '../../../../../utilities/tests/storybook';
+import { richTextMentionUsersViewTag } from '..';
+import {
+    disabledStates,
+    type DisabledState
+} from '../../../../../utilities/tests/states';
+import {
+    bodyFont,
+    bodyFontColor
+} from '../../../../../theme-provider/design-tokens';
+
+const metadata: Meta = {
+    title: 'Tests/Rich Text Mention: User',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const component = ([
+    disabledName,
+    disabled
+]: DisabledState): ViewTemplate => html`
+    <style class='code-hide'>
+        .mention-container {
+            font: var(${bodyFont.cssCustomProperty});
+            color: var(${bodyFontColor.cssCustomProperty});
+            margin: 4px;
+        }
+    </style>
+    <div class="mention-container">
+        <${richTextMentionUsersViewTag} mention-href="user:1" mention-label="John Doe" ?disabled="${() => disabled}" disable-editing>
```

Comment:
for both of these can you put some text before and after so that one can test out selection across the control.
Maybe something like:
```
[Mention View]@John Doe[Mention View Disabled]
```